<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layersAdaptDialogBase</class>
 <widget class="QDialog" name="layersAdaptDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>614</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Projektdatei anpassen</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Anpassung starten ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>Anpassung starten ...</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_selectLayers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>QKan-Layer</string>
   </property>
   <widget class="QRadioButton" name="rb_adaptSelected">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;nur aktivierte Layer berücksichtigen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>nur aktivierte Layer berücksichtigen</string>
    </property>
    <property name="text">
     <string>ausgewählte anpassen</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_adaptAll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;alle QKan-Layer bearbeiten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>alle QKan-Layer bearbeiten</string>
    </property>
    <property name="text">
     <string>alle anpassen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_completeLayers">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fehlende QKan-Layer mit Geodaten entsprechend dem oben gewählten QKan-Standard hinzuladen. Hinweis: Attributtabellen werden auf jeden Fall hinzugeladen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Fehlende QKan-Geodaten-Layer entsprechend dem oben gewählten QKan-Standard hinzuladen</string>
    </property>
    <property name="text">
     <string>Fehlende QKan-Geodaten-
Layer hinzuladen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_LayersAdapt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>420</width>
     <height>190</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Layer anpassen</string>
   </property>
   <widget class="QCheckBox" name="cb_adaptForms">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formularanbindung für die gewählten QKan-Layern wiederherstellen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formularanbindung für die gewählten QKan-Layern wiederherstellen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Formularanbindungen auf QKan-Standard setzen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_adaptTableLookups">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wertebeziehungen in den gewählten QKan-Layern neu setzten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Wertebeziehungen in den gewählten QKan-Layern neu setzten</string>
    </property>
    <property name="text">
     <string>Wertbeziehungungen in Tabellen aus QKan-Vorlage anpassen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_adaptKBS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Projektionssystem für gewählte Layer anpassen. Bei aktiver Datenbankanbindung immer aktiv!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Projektionssystem für gewählte Layer anpassen. Bei aktiver Datenbankanbindung immer aktiv!</string>
    </property>
    <property name="text">
     <string>Projektionssystem an Datenquelle anpassen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_adaptDB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gewählte Layer werden mit angegebener Datenbank verknüpft. Anpassung Projektionssystem wird automatisch aktiviert!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Gewählte Layer werden mit angegebener Datenbank verknüpft. Anpassung Projektionssystem wird automatisch aktiviert!</string>
    </property>
    <property name="text">
     <string>Datenbankanbindung der Layer anpassen (*.sqlite):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_selectQKanDB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Suchfenster öffnen ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Suchfenster öffnen ...</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tf_qkanDB">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aktuelle QKan - Datenbank&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Aktuelle QKan - Datenbank</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_zoomAll">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nach der Bearbeitung wird der Zoom-Maßstab auf alle geladenen Inhalte angepasst&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Nach der Bearbeitung wird der Zoom-Maßstab auf alle geladenen Inhalte angepasst</string>
    </property>
    <property name="text">
     <string>Zoom auf alle Layer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_projectTemplate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>420</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string comment="Nach dem Import eine Projektdatei neu erzeugen und speichern (optional)"/>
   </property>
   <property name="title">
    <string>Auswahl der Vorlage</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lb_projectTemplate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Einstellungen aus Projektdatei übernehmen (*.qgs):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_selectProjectTemplate">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>30</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Suchfenster öffnen ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Suchfenster öffnen ...</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tf_projectTemplate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Layereinstellungen aus gewählter Projektdatei übernehmen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Layereinstellungen aus gewählter Projektdatei übernehmen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_applyQKanTemplate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QKan-Standard für die nachfolgenden Anpassungen anwenden&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>QKan-Standard für die nachfolgenden Anpassungen anwenden</string>
    </property>
    <property name="text">
     <string>QKan-Standard verwenden</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_setNodeTypes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>420</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Auswertung Knotentypen</string>
   </property>
   <widget class="QCheckBox" name="cb_updateNodetype">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In der Tabelle &amp;quot;schaechte&amp;quot; Spalte &amp;quot;knotentyp&amp;quot; erzeugen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>In der Tabelle &quot;schaechte&quot; Spalte &quot;knotentyp&quot; erzeugen</string>
    </property>
    <property name="text">
     <string>Knotentypen in Tabelle eintragen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_updateQkanDB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>420</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>QKan-Datenbank</string>
   </property>
   <widget class="QCheckBox" name="cb_qkanDBUpdate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Struktur der QKan-Datenbank wird auf den aktuellen Stand angepasst. Dabei werden auch die Eingabeformulare aktualisiert.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Struktur der QKan-Datenbank wird auf den aktuellen Stand angepasst.</string>
    </property>
    <property name="text">
     <string>auf aktuelle Version anpassen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>layersAdaptDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>layersAdaptDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
